### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by fernandpeng**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,25 +22,32 @@ Google AdMob Integration:
...
-1. [CocoaPods](https://cocoapods.org) is a dependency manager for Cocoa projects. For usage and installation instructions, visit their website.
- Before running the project you have to install the cocoapods dependecy to the project by running bellow command using terminal in root folder of desired Sample App (Swift or Objective C)
+_**CocoaPods** is a dependency manager for Cocoa projects and it's required for this project. For usage and installation instructions, visit [their official documentation](https://guides.cocoapods.org/)._
 
-```bash
-pod install
-```
+1. Install the cocoapods dependecy to the project by running bellow command using terminal in root folder of desired language:
+ 	 - [Swift](https://github.com/Ogury/ogury-sdk-sampleapp-ios/tree/master/Swift "Swift")
+	  - [Objective C](https://github.com/Ogury/ogury-sdk-sampleapp-ios/tree/master/Swift "Objective C")
```
Comment (Pernic) [inline README.md]: remark (blocking) : https://github.com/Ogury/ogury-sdk-sampleapp-ios/tree/master/Swift "Objective C" are you sure

**Review by Pernic — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (fernandpeng) [reply README.md]: Fixed in 9cbb815

**Review by Pernic — APPROVED** [review]

**Comment by jesphinpt** [comment]
Can anyone clarify this

How to support Ogury bidding source as part of AdMob Mediation Adapter in Applovin?

Steps:

- I am using Applovin mediation adapters in my app.
- Google AdMob mediation is available as part of the mediation adapters.
- I would like to add Ogury bidding source as part of Google AdMob
- Should I add Ogury Ads SDK to my app or is it not needed to support this integration.

**Comment by MarcoVieiraAntunes** [comment]
Ogury can be used directly through Applovin's MAX mediation since we're a supported bidder - there is no need to use MAX mediating AdMob mediating Ogury.
The adapter is maintained by them, and can be found here: https://dash.applovin.com/documentation/mediation/ios/mediation-adapters?network=OGURY_PRESAGE_NETWORK .

**Comment by jesphinpt** [comment]
If I add Ogury SDK to my app i am getting the below errors


```
Showing Recent Errors Only
Undefined symbol: _OBJC_CLASS_$_OGCInternal
Undefined symbol: _OBJC_CLASS_$_OguryAbstractLogMessage
Undefined symbol: _OBJC_CLASS_$_OguryLog
Undefined symbol: _OBJC_CLASS_$_OguryNetworkClient
Undefined symbol: _OBJC_CLASS_$_OguryNetworkRequestBuilder
Undefined symbol: _OBJC_CLASS_$_OguryOSLogger
Undefined symbol: _OBJC_CLASS_$_OguryPersistentEventBus
Undefined symbol: _OBJC_METACLASS_$_OguryAbstractLogMessage

```

**Comment by MarcoVieiraAntunes** [comment]
First can you verify you imported manually all the below:

**OgurySdk.xcframework**
**OguryAds.xcframework
OguryChoiceManager.xcframework
OguryCore.xcframework
OMSDK_Ogury.xcframework** (it can be found in the OguryAds zip file)
 
Second, can you check that all frameworks are on mode "Do Not Embed":
 
Screenshot 2023-03-28 at 15 54 32

Third, can you add a -ObjC flag in the build settings:

Screenshot 2023-03-28 at 15 54 17

Please let me know how it goes.

**Comment by jesphinpt** [comment]
@MarcoVieiraAntunes I didn't found these all frameworks. Could you provide any zip for the above framework.

**Comment by MarcoVieiraAntunes** [comment]
OgurySdk.xcframework
OguryAds.xcframework
OguryChoiceManager.xcframework
OguryCore.xcframework
OMSDK_Ogury.xcframework (it can be found in the OguryAds zip file)

https://binaries.ogury.co/release/ios/4.1.1/OgurySdk-4.1.1.tar.gz
https://binaries.ogury.co/release/ads-ios/3.4.1/OguryAds-3.4.1.tar.gz
https://binaries.ogury.co/release/choice-manager-ios/4.1.0/OguryChoiceManager-4.1.0.tar.gz
https://binaries.ogury.co/release/core-ios/1.3.0/OguryCore-1.3.0.tar.gz

**Comment by jesphinpt** [comment]
The above works after adding the sdk's. Could you please let me know the above are the latest version frameworks of Ogury?
 
OguryAds-3.4.1 is 3.4.1
[OguryChoiceManager-4.1.0.tar.gz]
OguryCore 1.3.0

If you don't mind could you let mw know from where you get these links? I didn't found these like doc in the Ogury https://ogury-ltd.gitbook.io/ios/

**Comment by jesphinpt** [comment]
@MarcoVieiraAntunes Could you please provide the OguryAdmobMediation Adapter as well. It would be helpful for us.

**Comment by MarcoVieiraAntunes** [comment]
https://ogury-ltd.gitbook.io/release-notes/ios/ogury-adapters/admob

**Comment by jesphinpt** [comment]
I have the GoogleMobileAdsSdk already. I am asking there is OguryMediationGoogleMobileAds framework which i need to download manually and add it to my Xcode libraries. 

I can't able to get this OguryMediationGoogleMobileAds framework from the Ogury.

**Comment by MarcoVieiraAntunes** [comment]
https://binaries.ogury.co/release/mediation-google-mobiles-ads-ios/4.1.1.0/OguryMediationGoogleMobileAds-4.1.1.0.tar.gz

**Comment by jesphinpt** [comment]
@MarcoVieiraAntunes We are getting the below errors after adding the OguryMediationGoogleAds.framework

The build failed with below runtime error:

```
Undefined symbols for architecture x86_64:
  "_SKStoreProductParameterAdNetworkSourceIdentifier", referenced from:
      -[OGAStoreKitState loadStoreKitViewController:] in OguryAds(OGAStoreKitState.o)
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)

```
The same got failed while Archive with the below error:

```
OguryAds(OGAAdManager.o)
 "_objc_msgSend$systemTimeZone", referenced from:
   -[OGAAdServerMonitorRequestBuilder timeZone] in OguryAds(OGAAdServerMonitorRequestBuilder.o)
 "_objc_msgSend$initWithAdUnitId:bannerView:delegateDispatcher:adManager:notificationCenter:monitoringDispatcher:log:", referenced from:
   -[OGABannerAdInternalAPI initWithAdUnitId:bannerView:delegateDispatcher:] in OguryAds(OGABannerAdInternalAPI.o)
 "_objc_msgSend$objectForKeyedSubscript:", referenced from:
   -[OGWMonitoringInfo assetKey] in OgurySdk(OGWMonitoringInfo.o)
   -[OGWMonitoringInfo getValueForKey:] in OgurySdk(OGWMonitoringInfo.o)
   -[OGWMonitoringInfo putAll:] in OgurySdk(OGWMonitoringInfo.o)
   -[OGWMonitoringInfo containsAll:] in
OgurySdk(OGWMonitoringInfo.o)
   -[OGWMonitoringInfoFetcher populateAppVersion:] in OgurySdk(OGWMonitoringInfoFetcher.o)
   -[OGWMonitoringInfoSerializer deserialize:] in OgurySdk(OGWMonitoringInfoSerializer.o)
   -[OGAAdMonitorEvent asDisctionary] in OguryAds(OGAAdMonitorEvent.o)
   ...
 "_objc_msgSend$closeButtonBase64", referenced from:
   -[OGAMRAIDAdDisplayer setupViews] in OguryAds(OGAMRAIDAdDisplayer.o)
 "_objc_msgSend$appendMonitoringInfoAndSendIfNecessary:", referenced from:
   -[OGWWrapper startWithConfiguration:] in OgurySdk(OGWWrapper.o)
 "_objc_msgSend$hasPrefix:", referenced from:
   +[OGASKAdNetworkService sdkIsCompatibleWithSKAdNetwork] in OguryAds(OGASKAdNetworkService.o)
   -[OGAJSONModelError errorByPrependingKeyPathComponent:] in OguryAds(OGAJSONModelError.o)
   +[UIColor(OGAString) colorFromString:] in OguryAds(UIColor+OGAString.o)
   -[OGYJSONModelError errorByPrependingKeyPathComponent:] in OguryChoiceManager(OGYJSONModelError.o)
 "_objc_msgSend$closeWebView:", referenced from:
   -[OGAMraidCommandsHandler handleMraidCommand:] in OguryAds(OGAMraidCommandsHandler.o)
 "_objc_msgSend$omidSDK", referenced from:
   -[OGAOMIDService isOMIDActive] in OguryAds(OGAOMIDService.o)
   -[OGAOMIDService activateOMID] in OguryAds(OGAOMIDService.o)
 "_objc_msgSend$initWithBundle:", referenced from:
   -[OGAViewControllerOrientationHelper init] in OguryAds(OGAViewControllerOrientationHelper.o)
 "_objc_msgSend$eventNameFromShowErrorEvent:", referenced from:
   -[OGAMonitoringDispatcher sendShowErrorEvent:sessionId:adUnitId:] in OguryAds(OGAMonitoringDispatcher.o)
   -[OGAMonitoringDispatcher sendShowErrorEventAdExpired:sessionId:adUnitId:] in OguryAds(OGAMonitoringDispatcher.o)
 "_objc_msgSend$blackListViewControllers", referenced from:
   -[OGAAdConfiguration copyWithZone:] in OguryAds(OGAAdConfiguration.o)
   -[OGAThumbnailAdViewController viewControllerListUpdated] in OguryAds(OGAThumbnailAdViewController.o)
 "_objc_msgSend$name", referenced from:
   -[OGACloseAdAction performAction:error:] in OguryAds(OGACloseAdAction.o)
   -[OGAAdContainer state] in OguryAds(OGAAdContainer.o)
   -[OGAOpenStoreKitAction performAction:error:] in OguryAds(OGAOpenStoreKitAction.o)
   -[OGAUnloadAdAction performAction:error:] in OguryAds(OGAUnloadAdAction.o)
   -[OGADevice init] in OguryAds(OGADevice.o)
   -[OGADevice mapped] in OguryAds(OGADevice.o)
   -[OGAAdExposureController didReceiveKeyboardVisibilityDidChangeNotification:] in OguryAds(OGAAdExposureController.o)
   ...
 "_objc_msgSend$hasSentDisplayedDelegate", referenced from:
   -[OguryBannerAdDelegateDispatcher displayed] in OguryAds(OguryBannerAdDelegateDispatcher.o)
   -[OguryAdsOptinVideoDelegateDispatcher displayed] in OguryAds(OguryAdsOptinVideoDelegateDispatcher.o)
   -[OguryAdsThumbnailAdDelegateDispatcher displayed] in OguryAds(OguryAdsThumbnailAdDelegateDispatcher.o)
   -[OguryInterstitialAdDelegateDispatcher displayed] in OguryAds(OguryInterstitialAdDelegateDispatcher.o)
   -[OguryOptinVideoAdDelegateDispatcher displayed] in OguryAds(OguryOptinVideoAdDelegateDispatcher.o)
   -[OguryAdsBannerDelegateDispatcher displayed] in OguryAds(OguryAdsBannerDelegateDispatcher.o)
   -[OguryAdsInterstitialDelegateDispatcher displayed] in OguryAds(OguryAdsInterstitialDelegateDispatcher.o)
   ...
 "_objc_msgSend$setExpirationSource:", referenced from:
   -[OGAAdControllerFactory createControllerForAd:sequence:configuration:] in OguryAds(OGAAdControllerFactory.o)
   -[OGAExpirationContext initFrom:withExpirationTime:adPosition:] in OguryAds(OGAExpirationContext.o)
 "_objc_msgSend$storeProductViewController", referenced from:
   -[OGAStoreKitState loadStoreKitViewController:] in OguryAds(OGAStoreKitState.o)
   -[OGAStoreKitState display:error:] in OguryAds(OGAStoreKitState.o)
 "_objc_msgSend$sendExpirationTrackerEventForAd:", referenced from:
   -[OGAAdController isExpired] in OguryAds(OGAAdController.o)
 "_objc_msgSend$continueLoadAdAfterAdContentsPrepared:ads:error:", referenced from:
   ___54-[OGAAdManager continueLoadAdAfterAdSynced:ads:error:]_block_invoke in OguryAds(OGAAdManager.o)
 "_objc_msgSend$setIsStandardJSONType:", referenced from:
   -[OGAJSONModel __inspectProperties] in OguryAds(OGAJSONModel.o)
   -[OGYJSONModel __inspectProperties] in OguryChoiceManager(OGYJSONModel.o)
 "_objc_msgSend$integerForKey:", referenced from:
   -[OGWWrapper registerAttributionForSKAdNetwork] in OgurySdk(OGWWrapper.o)
 "_objc_msgSend$setActive:", referenced from:
   -[OGAMRAIDAdDisplayer deactivateConstrainIfFound:inView:] in OguryAds(OGAMRAIDAdDisplayer.o)
   -[OGAMRAIDAdDisplayer replaceOrAddConstraint:inView:] in OguryAds(OGAMRAIDAdDisplayer.o)
 "_objc_msgSend$viewProvider", referenced from:
   -[OGABannerAdContainerState display:error:] in OguryAds(OGABannerAdContainerState.o)
   -[OGAAdControllerFactory addTransitionsForBannerAd:configuration:builder:] in OguryAds(OGAAdControllerFactory.o)
   -[OGAAdConfiguration copyWithZone:] in OguryAds(OGAAdConfiguration.o)
 "_objc_msgSend$setIsCloseButtonHidden:", referenced from:
   -[OGAMRAIDAdDisplayer setupViews] in OguryAds(OGAMRAIDAdDisplayer.o)
   -[OGAMRAIDAdDisplayer setCloseButtonAsHidden:withDelay:] in OguryAds(OGAMRAIDAdDisplayer.o)
 "_objc_msgSend$setCampaignId:", referenced from:
   -[OGAAdMonitorEvent initWithTimestamp:sessionId:eventCode:event:dispatchType:adUnitId:campaignId:creativeId:detailsDictionary:errorType:errorContent:] in OguryAds(OGAAdMonitorEvent.o)
   -[OGABannerAdInternalAPI loadWithCampaignId:creativeId:size:] in OguryAds(OGABannerAdInternalAPI.o)
   -[OGABannerAdInternalAPI loadWithAdMarkup:size:] in OguryAds(OGABannerAdInternalAPI.o)
   -[OGAInterstitialAdInternalAPI loadWithCampaignId:creativeId:] in OguryAds(OGAInterstitialAdInternalAPI.o)
   -[OGAInterstitialAdInternalAPI loadWithAdMarkup:] in OguryAds(OGAInterstitialAdInternalAPI.o)
   -[OGAOptinVideoAdInternalAPI loadWithCampaignId:creativeId:] in OguryAds(OGAOptinVideoAdInternalAPI.o)
   -[OGAOptinVideoAdInternalAPI loadWithAdMarkup:] in OguryAds(OGAOptinVideoAdInternalAPI.o)
   ...
 "_objc_msgSend$setPreviousParentViewControllerBounds:", referenced from:
   -[OGABannerAdContainerState observeValueForKeyPath:ofObject:change:context:] in OguryAds(OGABannerAdContainerState.o)
 "_objc_msgSend$setExpirationDate:", referenced from:
   -[OGCInstanceToken removeExpirationDate] in OguryCore(OGCInstanceToken.o)
   -[OGCInstanceToken updateExpirationDate] in OguryCore(OGCInstanceToken.o)
 "_objc_msgSend$keyMapper", referenced from:
   -[OGAJSONModel __setup__] in OguryAds(OGAJSONModel.o)
   -[OGYJSONModel __setup__] in OguryChoiceManager(OGYJSONModel.o)
 "_objc_msgSend$destroy", referenced from:
   -[OguryAdsBanner close] in OguryAds(OguryAdsBanner.o)
   -[OguryBannerAd destroy] in OguryAds(OguryBannerAd.o)
 "_objc_msgSend$dateByAddingTimeInterval:", referenced from:
   -[OGAAdController isExpired] in OguryAds(OGAAdController.o)
   +[OGAConsentEventBus hasExpired:] in OguryAds(OGAConsentEventBus.o)
   -[OGAProfigManager isProfigExpired] in OguryAds(OGAProfigManager.o)
   ___49-[OGAMraidAdWebView startMRAIDProcessForContent:]_block_invoke.16 in OguryAds(OGAMraidAdWebView.o)
   ___49-[OGAMraidAdWebView startMRAIDProcessForContent:]_block_invoke.23 in OguryAds(OGAMraidAdWebView.o)
   -[OGAWebViewCleanupManager cleanUpObject:] in OguryAds(OGAWebViewCleanupManager.o)
   +[OguryConsentEventBus hasExpired:] in OguryChoiceManager(OguryConsentEventBus.o)
   ...
 "_objc_msgSend$setBlackListedTracks:", referenced from:
   -[OGAProfigManager updateMonitoringTrackingAndBlacklistedTracks:] in OguryAds(OGAProfigManager.o)
 "_objc_msgSend$initWithEventURL:networkClient:log:", referenced from:
   -[OGAMetricsServiceSendCustomEventOperation initWithEventURL:] in OguryAds(OGAMetricsServiceSendCustomEventOperation.o)
 "_objc_msgSend$setTimeoutIntervalForRequest:", referenced from:
   -[OguryNetworkClient init] in OguryCore(OguryNetworkClient.o)
 "_objc_msgSend$initWithRequestBuilder:networkClient:persistanceStore:log:", referenced from:
   -[OGMServerMonitor initWithRequestBuilder:persistanceStore:] in OguryAds(OGMServerMonitor.o)
 "_objc_msgSend$setIsAskOrEditCalled:", referenced from:
   -[OguryChoiceManager resetSetup] in OguryChoiceManager(OguryChoiceManager.o)
   -[OguryChoiceManager askWithViewController:andCompletionBlock:] in OguryChoiceManager(OguryChoiceManager.o)
   -[OguryChoiceManager editWithViewController:andCompletionBlock:] in OguryChoiceManager(OguryChoiceManager.o)
 "_objc_msgSend$adUnit", referenced from:
   -[OGAMraidBrowserWebView loadWebViewWithCommandforBrowser:] in OguryAds(OGAMraidBrowserWebView.o)
   -[OGAAdExpirationManager sendExpirationTrackerEventForAd:] in OguryAds(OGAAdExpirationManager.o)
   -[OGAThumbnailAdViewController viewDidLoad] in OguryAds(OGAThumbnailAdViewController.o)
   -[OGAMraidAdWebView loadWithContent:] in OguryAds(OGAMraidAdWebView.o)
   -[OGAMraidAdWebView startMRAIDProcessForContent:] in OguryAds(OGAMraidAdWebView.o)
   +[OGAAdParser shouldParseAd:withConfiguration:] in OguryAds(OGAAdParser.o)
   -[OGAMRAIDAdDisplayer rewardWasReceived] in OguryAds(OGAMRAIDAdDisplayer.o)
   ...
 "_objc_msgSend$initWithAd:event:", referenced from:
   -[OGAAdController sendLoadedTracker] in OguryAds(OGAAdController.o)
   -[OGAAdExpirationManager sendExpirationTrackerEventForAd:] in OguryAds(OGAAdExpirationManager.o)
   -[OGAAdContentPreCacheManager sendLoadedErrorEventForAd:error:] in OguryAds(OGAAdContentPreCacheManager.o)
   -[OGAAdSequenceCoordinator controller:didUnLoadAd:] in OguryAds(OGAAdSequenceCoordinator.o)
   -[OGAAdSequenceCoordinator controller:didUnLoadWithNextAd:] in OguryAds(OGAAdSequenceCoordinator.o)
   -[OGAAdImpressionManager sendDefaultImpressionTracker:] in OguryAds(OGAAdImpressionManager.o)
 "_objc_msgSend$conformsToProtocol:", referenced from:
   -[OGWLogFormatter formatLogMessage:] in OgurySdk(OGWLogFormatter.o)
   -[OGALogFormatter formatLogMessage:] in OguryAds(OGALogFormatter.o)
   -[OguryGoogleAdapterLogFormatter formatLogMessage:] in OguryMediationGoogleMobileAds(OguryGoogleAdapterLogFormatter.o)
   -[OGCLogFormatter formatLogMessage:] in OguryCore(OGCLogFormatter.o)
   -[OGYLogFormatter formatLogMessage:] in OguryChoiceManager(OGYLogFormatter.o)
ld: symbol(s) not found for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

**Pull request merged**